### ISSUE TITLE
Suggestion: Ability to change item background color or opacity, or have a visible item outline.

### ISSUE THREAD

**Issue by ariosos** [mezz/JustEnoughItems#2172]
A lot of times, items that are very dark are easy to miss on the item list (the fact that JEI always displays 1/1 pages when there's no items is also confusing).

![image](https://user-images.githubusercontent.com/7917753/100000537-b4185980-2d8f-11eb-9a42-e72ea27a517f.png)

**Comment by jursamaj** [comment]
Agreed.  Actually, I think there should be an option for a background, like the bookmark side has.  I just found that in the options tonight, and am amazed there is no such option for the search side.

**Comment by mezz** [comment]
Thanks for the suggestion!

This was implemented in JEI 9.4.2 for Minecraft 1.18.2:
* https://github.com/mezz/JustEnoughItems/pull/2711

The ingredient-list config settings have a “Draw GUI Background” option that draws a background and slot texture behind the items.
